plugins/dotnet/dotnet.plugin.sh

```diff
@@ -0,0 +1,40 @@
+# This scripts is copied from (MIT License):
```

Comment:
We add the following header line to every plugin file.

```suggestion
#! bash oh-my-bash.module
#
# This scripts is copied from (MIT License):
```

Also, there was a link after this line in the upstream OMZ, but the link is dropped in this PR. Then, there is nothing after "from". Please remove this line or add a link to complete the sentence.